README.md

```diff
@@ -126,6 +130,84 @@ Response:
   Body: <preimage_bytes>
 ```
 
+#### Admin Routes
+
+The proxy provides administrative endpoints to control runtime behavior. By default, these endpoints are disabled and must be explicitly enabled through configuration.
+
+To enable admin endpoints, set the `--admin-endpoints-enabled` flag to `true` or set the environment variable `EIGENDA_PROXY_ADMIN_ENDPOINTS_ENABLED=true` when starting the proxy server.
+
+When enabled, the following admin endpoints are available:
+
+```text
+Request:
+  GET /admin/v2-dispersal
+
+Response:
+  200 OK
+  Content-Type: application/json
+  Body: {"disperseToV2": boolean}
```

Comment:
longterm I'm curious if there's any value in changing this to something like `protocolVersion`

Reply:
[c7a17b9](https://github.com/Layr-Labs/eigenda-proxy/pull/360/commits/c7a17b9ccaea6e5c199aa8fd97d9c607dfc911fc)